Source/WebCore/css/CSSPrimitiveValue.cpp

```diff
@@ -682,8 +682,8 @@ double CSSPrimitiveValue::computeUnzoomedNonCalcLengthDouble(CSSUnitType primiti
         return ((propertyToCompute == CSSPropertyFontSize) ? fontDescription->specifiedSize() : fontDescription->computedSize()) * value;
     case CSSUnitType::CSS_EXS:
         ASSERT(fontMetrics);
-        if (fontMetrics->hasXHeight())
```

Comment:
Keeping the `Markable` API would allow 

```
if (fontMetrics->xHeight())
         return fontMetrics->xHeight * value
```


Is there an advantage with returning `std::optional<float>` ?

Reply:
Some font metrics are missing in a font, so I intended to use has_value() and value_or() of std::optional. But I found Markable also supported similar methods. I will replace std::optional<float> return values with Markable ones.

Do you mean `return fontMetrics->xHeight().value() * value`, by the way?